bundles/org.openhab.binding.kvv/README.md

```diff
@@ -13,14 +13,40 @@ This includes the name of the train, the final destination and the estimated tim
 
 Since every stop is represented by a KVV-provided id, this id has to be figured out via an API call.
 
-### Example Call for Stop 'Karlsruhe Volkswohnung'
+### Example Call for Stop 'Gottesauer Platz/BGV'
 
 ```bash
-# Request
-curl https://live.kvv.de/webapp/stops/byname/Volkswohnung\?key\=[APIKEY]
+export QUERY="gottesauer"
+curl https://www.kvv.de/tunnelEfaDirect.php?action=XSLT_STOPFINDER_REQUEST&name_sf=${QUERY}&outputFormat=JSON&type_sf=any
+```
 
-# Response
-{"stops":[{"id":"de:8212:72","name":"Karlsruhe Volkswohnung","lat":49.00381654,"lon":8.40393026}]}
+The exact `id` may be extracted from the JSON-encoded reponse. E.g.
+
+```json
+"points": [
+...
```

Comment:
These dots are rendered in a strange way, at least when viewing the rendered Markdown in GitHub. Can they be removed (here and at the end of provided example)?

Reply:
Also done.

Reply:
You have still not removed them, apparently.

Reply:
I am sorry, I forgot to push that :/. Should be done now.

Reply:
I don't understand th ediff rendering by GitHub but the rendering of the updated file looks good.